src/platform/icelake/include/platform/shim.h

```diff
@@ -233,6 +233,9 @@
 #define DMWBA_ENABLE		(1 << 0)
 #define DMWBA_READONLY		(1 << 1)
 
+/* DMIC power ON bit */
+#define DMIC_POWER_ON	0x01
```

Comment:
Notation for macros is to use BIT() macro now (some older ones still use a shift for bits).